orcestra_book/hera5.md

```diff
@@ -0,0 +1,183 @@
+
+```{raw-cell}
+!pip install easygems
+```
+
+## Loading the data from the catalog
+Open the catalog and select the *hierarchical HEALPix ERA5 data* `hera5` from the catalog.
+
+```{code-cell} ipython3
+cat = intake.open_catalog("https://tcodata.mpimet.mpg.de/internal.yaml")
+list(cat)
+```
+
+```{code-cell} ipython3
+cat.HERA5.to_dask()
+```
+
+## Some notes on the dataset
+### Dimensions
+* the dimensions are `time`, `cell` and (pressure-)`level` and the coordinate reference system `crs`. Here, `cell` refers to the cells in the HEALPix grid, which is a pixelation of a sphere. **Get to know more about HEALPix [here](https://easy.gems.dkrz.de/Processing/healpix/index.html#healpix).** The zoom level of the HERA5 data is $z = 7$ and the number of cells respectively $12 \cdot z^{4} = 196608$
+* you can distinguish between 2d and 3d variables by their dimensions, the former having only `(time, cell)` and the latter having `(time, level, cell)`.
+### Temporal hierarchy
+* The dataset is constructed hierarchically in time, which means that you can directly access either monthly aggregated values `P1M`, daily `P1D`, or the original hourly data `PT1H`. By default, the catalog entry points to the monthly values which are great to do a fast analysis of longterm and coarse features such as calculating the global mean surface temperature over 10 years. If you need higher temporal resolutions, you can add the respective acronym when loading the catalog: `ds = cat.HERA5(time="PT1H").to_dask()`.
+* In principle, one could also implement spatial hierarchies. However, the original ERA5 data is already rather coarse such that there is no need for coarser spatial (zoom) levels in the present case.
+### Spatial resolution
+The spatial resolution is given by the `zoom` level $z = 7$ which is closest to the original ERA5 model output and roughly corresponds to 0.35 x 0.7° effective grid resolution.
+### Variables
+Most variables are from the ERA5 "analysis" product. Only few variables such as `total precipitation` and `boundary layer height` are added from the "forecast" product. Each variable has a `type` attribute which will let you know whether it is from the `analysis` or `forecast` product.
+
++++
+
+## Plotting examples
+
+The below plotting examples show some typical usecases and the respective [`healpy`](https://healpy.readthedocs.io/en/latest/index.html) functions that are frequently used in the analysis.
+
+In the following, we will need to specify the `nest` parameter, which defines the arrangement of the cells in the HEALPix grid. We can extraxt this info from the dataset attributes with a little helper function.
+Also, we will need to spcify the `nside` parameter which defines the resolution of the grid by it's number of divisions along the side of a base-resolution pixel.
```

Comment:
This paragraph might be more useful if it would point to the `easygems` package that now includes the respective functions.

Reply:
I removed the paragraph completely. I think this streamlines the example and helps users to focus on the actual examples. If someone is interested in these parameters, they can look into the HEALPix/healpy examples that are linked at the top.